include/api/CInferenceModelDefinition.h

```diff
@@ -47,6 +47,17 @@ class API_EXPORT CSerializableToJsonStream {
     virtual void addToJsonStream(TGenericLineWriter& /*writer*/) const = 0;
 };
 
+class API_EXPORT CSerializableToJsonDocumentCompressed : public CSerializableToJsonStream{
+    public:
+        using TRapidJsonWriter = core::CRapidJsonConcurrentLineWriter;
+    public:
+    virtual ~CSerializableToJsonDocumentCompressed() = default;
+    virtual void addToDocumentCompressed(TRapidJsonWriter& writer) const = 0;
+    virtual void addToDocumentCompressed(TRapidJsonWriter& writer, std::string compressed_doc_tag) const;
```

Comment:
```suggestion
    virtual void addToDocumentCompressed(TRapidJsonWriter& writer, std::string compressedDocTag) const;
```

Reply:
whoops. Came from the string constant and I overlooked the naming. Fixed now.